### PR TITLE
pull request reminder to update crate version

### PR DESCRIPTION
**Pull request by NickLarsenNZ**

### CODE REVIEW

**Comment by sbernauer** [comment]
I have to admit I don't like yet another additional checkbox (we don't even stick to the current ones) and would prefer to force all of us to install `serayuzgur.crates` in VSC. Also there are some crates which we cant update easily, such as `built`

**Comment by NickLarsenNZ** [comment]
> I have to admit I don't like yet another additional checkbox (we don't even stick to the current ones) and would prefer to force all of us to install `serayuzgur.crates` in VSC. Also there are some crates which we cant update easily, such as `built`

Yah, this was basically to stop me/us forgetting to bump the operator-rs crate version. But it is only that crate which was a problem, so maybe we can automate it?

I have raised a discussion around whether we want to keep these check lists. It would be nice to automate as much as possible, and only have a small list.

File: template/.github/pull_request_template.md [hunk 1]
```diff
@@ -24,6 +24,7 @@
 - [ ] (Integration-)Test cases added
 - [ ] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/style-guide).
 - [ ] Changelog updated
+- [ ] Cargo.toml version updated
```
[the remark this answers is not in the document]
Comment (NickLarsenNZ) [reply template/.github/pull_request_template.md:27]: @sbernauer shall we just add this checkbox to operator-rs?
Or are you strongly opposed (in which case I will close this PR)?

[the remark this answers is not in the document]
Comment (sbernauer) [reply template/.github/pull_request_template.md:27]: operator-rs would be fine for me, but we should make it a bit more expressive, such as

> When releasing: operator-rs version in Cargo.toml as well as changelog bumped

On the other hand that is basically all you need to do for a release xD

**Pull request closed without merging**